src/test/java/uk/gov/hmcts/pdda/business/services/formatting/MergeListUtilsTest.java

```diff
@@ -0,0 +1,101 @@
+package uk.gov.hmcts.pdda.business.services.formatting;
+
+import org.easymock.EasyMockExtension;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.io.IOException;
+import javax.xml.xpath.XPathExpression;
+import javax.xml.xpath.XPathExpressionException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * <p>
+ * Title: MergeListUtils Test.
+ * </p>
+ * <p>
+ * Description:
+ * </p>
+ * <p>
+ * Copyright: Copyright (c) 2024
+ * </p>
+ * <p>
+ * Company: CGI
+ * </p>
+ * 
+ * @author Luke Gittins
+ */
+@ExtendWith(EasyMockExtension.class)
+class MergeListUtilsTest {
+
+    private static final String EQUAL = "Result is Equal";
+    private static final String FALSE = "Result is False";
+
+    private static final String[] DAILYLISTROOTNODES = {"DailyList/CourtLists"};

```

Comment:
Change this in accordance with constant naming conventions: DAILY_LIST_ROOT_NODES along with the rest of the declarations.

Reply:
Fixed, changed this and the other variables to align with naming convention